src/foam/nanos/auth/resetPassword/ResendView.js

```diff
@@ -92,11 +92,11 @@ foam.CLASS({
   ],
 
   messages: [
-    { name: 'Instructions', message: "We've sent the instructions to your email. Please check your inbox to continue."}
+    { name: 'Instructions', message: 'We\'ve sent the instructions to your email. Please check your inbox to continue.'}
```

Comment:
The style guideline isn't to always use single-quotes, it's to use single quotes by default, unless it makes sense to use double quotes for reasons like avoiding escaping strings which include single-quotes.

Reply:
Resolved.